homeassistant/components/faadelays/binary_sensor.py

```diff
@@ -0,0 +1,94 @@
+"""Platform for FAA Delays sensor component."""
+from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.const import ATTR_ICON, ATTR_NAME
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, FAA_BINARY_SENSORS
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up a FAA sensor based on a config entry."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
+
+    binary_sensors = []
+    for kind, attrs in FAA_BINARY_SENSORS.items():
+        name = attrs[ATTR_NAME]
+        icon = attrs[ATTR_ICON]
+
+        binary_sensors.append(
+            FAABinarySensor(coordinator, kind, name, icon, entry.entry_id)
+        )
+
+    async_add_entities(binary_sensors, True)
```

Comment:
Do we need to refresh the coordinator data again during entity addition by passing the second argument `True`?